packages/readiness/vendir.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: vendir.k14s.io/v1alpha1
+kind: Config
+minimumRequiredVersion: 0.12.0
+directories:
+  - path: bundle/config/upstream
+    contents:
+      - path: crds/core
```

Comment:
readiness API exists only in the core API group, unlike featuregates and capabilities where their API definitions exist in multiple API groups, so we can vendir the readiness related CRDs into `crds` directory directly

Reply:
Updated path to `crd`